api/admin/controller/__init__.py

```diff
@@ -2584,3 +2583,161 @@ def validate_language_code(self, settings):
     def _is_language(self, language):
         # Check that the input string is in the list of recognized language codes.
         return LanguageCodes.string_to_alpha_3(language)
+
+
+class SitewideRegistrationController(SettingsController):
```

Comment:
This class used to be in sitewide_registration.py, which made it look like a controller that did something specific and needed its own route. As far as Adriana and I can tell, it doesn't do anything specific -- it's just a superclass for classes like `MetadataServicesController`. So I moved it to `__init__.py` with the rest of the helpful superclasses.

I hope that eventually this class can be simplified by replacing the registration dance with `RemoteRegistry.push`, similar to what we did with `DiscoveryServiceLibraryRegistrationsController`, but I decided not to tackle that with this branch.